tests/test_scripts.py

```diff
@@ -96,6 +139,23 @@ def test_import_prepHistFactory_and_cls(tmpdir, script_runner):
         assert 'CLs_exp' in d
 
 
+@pytest.mark.parametrize("backend", ["numpy", "tensorflow", "pytorch", "jax"])
+def test_fit_backend_option(tmpdir, script_runner, backend):
+    temp = tmpdir.join("parsed_output.json")
+    command = "pyhf xml2json validation/xmlimport_input/config/example.xml --basedir validation/xmlimport_input/ --output-file {0:s}".format(
+        temp.strpath
+    )
+    ret = script_runner.run(*shlex.split(command))
+
+    command = "pyhf fit --backend {0:s} {1:s}".format(backend, temp.strpath)
+    ret = script_runner.run(*shlex.split(command))
+
+    assert ret.success
+    ret_json = json.loads(ret.stdout)
+    assert ret_json
+    assert "mle_parameters" in ret_json
```

Comment:
we'll also need a slightly fleshed out test to assert the particular output structure to make sure we don't break things in the future for users who rely on the output of this CLI

Reply:
@matthewfeickert this coverage is needed.

Reply:
Okay. Can you change your review status in the future so if I miss these after you've given your review it shows up in the check at the end?